hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -807,6 +807,50 @@ def _track_subtype(self, init_values):
                 return self.env.ref("hr_timesheet_sheet.mt_timesheet_approved")
         return super()._track_subtype(init_values)
 
+    # ------------------------------------------------
+    # to prevent sending emails to the reviewer
+    # ------------------------------------------------
+
+    def _notify_record_by_email(
+        self,
+        message,
+        recipients_data,
+        msg_vals=False,
+        model_description=False,
+        mail_auto_delete=True,
+        check_existing=False,
+        force_send=True,
+        send_after_commit=True,
+        **kwargs
+    ):
+        """Overrides the _notify_record_by_email method,
```

Comment:
Do not change docstring of the original method.
Add a comment instead

Reply:
done